examples/calculator/calculator.algo.ts

```diff
@@ -33,12 +33,12 @@ class Calculator extends Contract {
    *
    * @returns The result of the operation
    */
-  doMath(a: uint64, b: uint64, operation: string): uint64 {
+  doMath(a: uint64, b: uint64, operation: utf8): uint64 {
```

Comment:
I wonder if `str` or `utf8string` is a clearer type name for this?

Reply:
I avoided `str` but I think it's a bit confusing since it's so close to `string`. I'm not opposed to `utf8string` but just went with `utf8` since it's shorter